src/main/resources/templates/fragments/footer.html

```diff
@@ -1,12 +1,11 @@
-<footer class="w-100 position-absolute z-1 bottom-0 bg-white">
-    <hr class="mt-0">
-    <div class="p-3 align-items-center justify-content-between d-flex">
+<footer class="position-fixed bottom-0 w-100 z-1 bg-white d-flex justify-content-center">
+    <div class="align-items-center justify-content-between d-flex border-top border-black w-100 h-100 p-4 mx-3">
         <a data-testid="swissuniversities-link" href="https://www.swissuniversities.ch/" target="_blank">
             <img data-testid="swissuniversities-logo" alt="Swissuniversities logo" height="22px" th:src="@{images/swissuniversities.png}">
         </a>
-        <div class="d-flex justify-content-end gap-4">
-            <a data-testid="impressum-link" class="link-primary" href="https://www.unilu.ch/impressum/" target="_blank">Impressum & Datenschutz</a>
-            <span th:text="${'Version ' + @environment.getProperty('spring.application.version')}"></span>
+        <div class="d-flex align-items-center gap-4">
+            <a class="link-primary" href="https://www.unilu.ch/impressum/" target="_blank" data-testid="impressum-link">Impressum & Datenschutz</a>
+            <p class="m-0" th:text="${'Version ' + @environment.getProperty('spring.application.version')}" data-testid="version-link"></p>
```

Comment:
```suggestion
            <p class="m-0" th:text="${'Version ' + @environment.getProperty('spring.application.version')}" data-testid="version"></p>
```
Maybe we should rename this to just `version` as it is no longer a link.